### PR TITLE
no panic logger

### PR DESCRIPTION
**Pull request by asa93**
wasm code was panicking if it was attempted to init several times which is not a critical issue

it created an error display that was scary on extension side

little change to fix it

### CODE REVIEW

File: crates/wasm/Cargo.toml [hunk 1]
```diff
@@ -54,6 +54,7 @@ remote-attestation-verifier = { workspace = true }
 p256 = { workspace = true }
 rand_core = { workspace = true }
 sha2 = { workspace = true }
+regex = "1.11.0"
```
Comment (saberistic) [inline crates/wasm/Cargo.toml:57]: is this part of this PR? I have this library included in the other PR

File: crates/wasm/src/lib.rs [hunk 2]
```diff
@@ -240,4 +240,49 @@ mod test {
 
         assert!(verifying_key.verify(&application_data, &signature).is_ok());
     }
+
+    #[test]
+    fn test_regex() {
```
Comment (saberistic) [inline crates/wasm/src/lib.rs:245]: is this a playground test? doesn't look like testing functions of the file

**Review by saberistic — CHANGES_REQUESTED** [review]
minor cleanup

File: crates/wasm/Cargo.toml (hunk 1 above)
Comment (asa93) [reply crates/wasm/Cargo.toml:57]: removed

File: crates/wasm/src/lib.rs (hunk 2 above)
Comment (asa93) [reply crates/wasm/src/lib.rs:245]: removed

**Comment by asa93** [comment]
changed made !

**Pull request merged**